### ISSUE TITLE
Removal of SC 3.2.7: Visible Controls from WCAG 2.2.

### ISSUE THREAD

**Issue by Ohadha-bobly** [w3c/wcag#3587]
Why was SC 3.2.7: Visible Controls scrapped from WCAG 2.2?

**Comment by alastc** [comment]
Draft response for review:
<hr> 

Visible Controls was in earlier drafts, but it was somewhere between difficult and impossible to define what would pass and fail such a criteria in a binary way.

Even relatively common interface examples (such as a play media button) can have complex states which makes testing difficult. Also, different people have different expactations of interfaces. There is no one-rule you can make which captures (or cuts through) the complexity.

We tested some examples, and there were enough that had very different results between testers that we couldn't continue. 

I think this was the last couple of meeting minutes on the topic: 
https://www.w3.org/2022/04/05-ag-minutes#item09
https://www.w3.org/2022/04/26-ag-minutes#t06

It is up for discussion again as part of WCAG 3, which should have other ways of evaluating interfaces to make it more possible.

**Comment by bruce-usab** [comment]
@Ohadha-bobly -- please close this issue if the "draft response for review" is enough of an explanation.

**Comment by alastc** [comment]
This response was approved by the group, so closing.
